### ISSUE TITLE
Reordering of tabs in dashboards

### ISSUE THREAD

**Issue by pascalberger** [Graylog2/graylog2-server#7470]
## What?

While it is possible to add multiple tabs on a dashboard, tabs can't be rearranged.

## Why?

Having the possibility to rearrange tabs would allow to add tabs to an existing dashboard and still keep them in a logical order.

## Your Environment

* Graylog Version: 3.2.1
* Elasticsearch Version: 6.8.6
* MongoDB Version: 3.6.17
* Operating System: Docker images
* Browser version:

**Comment by bit-plumber** [comment]
I agree.  It's frustrating when you come up with a great new use case but can't arrange it where it logically needs to be in the dashboard.

**Comment by bit-plumber** [comment]
I agree, this is a needed feature.

**Comment by xaviercho** [comment]
This Feature would be great as I'm currently remaking widgets in dashboards.

**Comment by r-duran** [comment]
Any update on this? When I add new pages (tabs) to existing dasboard, I can't re order them as my needs.

**Comment by JeroenStravers1** [comment]
I'll echo the above. I need quite a few tabs in alphabetical order for my use case and will need to add/remove entries. This is currently impossible. This forces me to make a single, gigantic dashboard instead, the tab function might as well not exist.

**Comment by chalfling** [comment]
@dennisoelkers Hello!
I updated from 4.2.9 to 4.3.0 and tab order was a mess. Why can't I reorder this?

**Comment by HiltiRaceteam** [comment]
We have the same problem after the update vom 4.2.9 to 4.3.0
The tab order is random, and if you duplicate a tab the order is recreated randomly.

**Comment by HungryHowies** [comment]
I'm Also having the same issue. with tab order with GL 4.3.1. If I duplicate the tab it will randomly  place it  either in front or behind the  original tab. If I create a new tab it seams I have no control of what order its placed.

**Comment by agamemnon23** [comment]
Since upgrading from 4.2.9 to 4.3.1, I'm also having the issue were all my dashboard tabs have been reordered.
Additionally, adding a new tab or duplicating a tab reorders all of them again.

**Comment by dennisoelkers** [comment]
Hey @chalfling, @HiltiRaceteam, @HungryHowies, @agamemnon23,

this is thanks for reporting this. We will look into it as soon as possible!

**Comment by H2Cyber** [comment]
I am also affected, as the update from 4.2.9 to 4.3.1 reordered all of my dashboard pages randomly. 

It would be great if we could have the ability to arrange dasjhboard pages, or if they get arranged alphabetically.

**Comment by dennisoelkers** [comment]
PR for `4.3` is merged now (https://github.com/Graylog2/graylog2-server/pull/12849). We are trying to cut a release including it as soon as possible, hopefully beginning of next week.
